Cut legends down (#285)
AlexsLemonade/ScPCA-manuscript: content/100.figure-table-legends.md

All single-cell or single-nuclei samples in the Portal are shown under the "All Samples" heading, a subset of which also have additional modalities as shown under the "Samples with additional modalities" heading.
As indicated, sample suspensions are either single-cell or single-nuclei.
For example, 75 single-cell samples and 101 single-nuclei samples have accompanying bulk RNA-seq data.
Note that two samples were sequenced with both single-cell and single-nuclei suspensions.
Samples with only bulk RNA-seq or spatial transcriptomics modalities are not shown.

C. Example of a project card as displayed on the "Browse" page of the ScPCA Portal and a "Visualize" view for a library within that project, colored by consensus cell type annotation.
This project card and visualized sample are from project `SCPCP000004` [@doi:10.1101/2024.01.07.574538; @doi:10.1186/s13059-024-03309-4].
Project cards include information about the number of samples, technologies and modalities, additional sample metadata information, submitter-provided diagnoses, and a submitter-provided abstract.
Where available, submitter-provided citation information and other databases hosting this data are also provided.

> The visualization employs the UCSC Cell Browser [@doi:10.1093/bioinformatics/btab503], enabling interactive exploration with user-selectable coloring options including cell types, gene-level expression, and other per-cell annotations created by `scpca-nf`.

Reviewer: ```suggestion
The visualization employs the UCSC Cell Browser [@doi:10.1093/bioinformatics/btab503], enabling interactive exploration with user-selectable coloring options for cell types, gene-level expression, and other per-cell annotations created by `scpca-nf`.
```

<br><br><br><br><br>

<!-- Figure 2 -->
![**Overview of the `scpca-nf` workflow and download file structures.**](https://raw.githubusercontent.com/AlexsLemonade/scpca-paper-figures/v0.2.2/figures/compiled_figures/pngs/figure_2.png?sanitize=true){#fig:fig2 tag="2" width="7in"}

A. Overview of `scpca-nf`, the primary workflow for processing single-cell/nuclei RNA-seq data for the ScPCA Portal.

> Mapping is first performed with `alevin-fry` to generate a gene-by-cell count matrix, which is read into `R` and converted into a `SingleCellExperiment` (`SCE`) object.

Reviewer: ```suggestion
Mapping is first performed with `alevin-fry` to generate a gene-by-cell count matrix, which is read into `R` and converted into a `SingleCellExperiment` (`SCE`) object (`Unfiltered SCE Object`).
```

> Next, empty droplets are filtered out.

Reviewer: ```suggestion
Next, empty droplets are filtered out (`Filtered SCE Object`).
```

The filtered object undergoes additional post-processing, including removing low-quality cells, normalizing counts, and performing dimension reduction with principal components analysis and UMAP.

> The object undergoes cell type annotation and CNV inference.

Reviewer: ```suggestion
The object undergoes cell type annotation and CNV inference (`Processed SCE Object`).
```

> Objects at different processing levels are exported as the workflow proceeds. 

Reviewer: ```suggestion
```

F. UMAP of log-normalized RNA expression colored by the number of genes detected.

G. UMAP of log-normalized RNA expression for the top four most variable genes, colored by the given gene's expression.
In the actual summary QC report, the top 12 most highly variable genes are shown.

H. File download structure for an ScPCA Portal project download in `SingleCellExperiment` (`SCE`) format.
The download folder is named according to the project ID, data format, and the date it was downloaded.
Download folders contain a folder for all single-cell data, `_single-cell`.
Here, each sample ID has a dedicated folder containing the three processing levels of the expression data, the summary QC report, and cell type report all named according to the ScPCA library ID.
The `single-cell_metadata.tsv` file contains sample metadata for all samples included in the download.
The `README.md` file provides information about the contents of each download file, additional contact and citation information, and terms of use for data downloaded from the ScPCA Portal.
The folder `_bulk` is only present for projects that also have bulk RNA-Seq data

> It contains a gene-by-sample matrix of raw gene expression quantified by `salmon` and associated metadata for samples with bulk RNA-Seq data.

Reviewer: Pretty sure it's counts but please do double check

```suggestion
The folder `_bulk` – only present for projects with bulk RNA-Seq data – contains a gene-by-sample matrix of counts quantified by `salmon` and associated metadata for samples with bulk RNA-Seq data.
```

A. Expanded view of cell type annotation process within `scpca-nf`, as introduced in Figure {@fig:fig2}A.
Cell type annotation is performed on the `Processed SCE Object`.
`SingleR` [@doi:10.1038/s41590-018-0276-y] annotation uses a `celldex` [@doi:10.1038/s41590-018-0276-y] reference dataset with ontology labels, `CellAssign`[@doi:10.1038/s41592-019-0529-1] annotation uses a list of marker genes compiled from `PanglaoDB` [@doi:10.1093/database/baz046], and `SCimilarity` [@doi:10.1038/s41586-024-08411-y] annotation uses the `SCimilarity` foundation model.
These cell typing results, along with consensus cell type annotations, are then added to the `Processed SCE Object`.
A cell type summary report with information about reference sources, comparisons among cell type annotation methods, and diagnostic plots is created.
Note that cell type annotations are also included in the `Processed AnnData Object` (Figure {@fig:fig2}A).

B. Diagram of ontology-aware consensus cell type annotation performed in `scpca-nf`, using T cell annotation as an example.
The numbers in parentheses indicate the number of descendants for each cell type in the Cell Ontology.

> Pairwise comparisons are made among annotations, and the consensus cell type is the latest common ancestor with the fewest descendants, indicated by the black check mark.

Reviewer: ```suggestion
Pairwise comparisons are made among annotations, and the consensus cell type is the latest common ancestor in Cell Ontology with the fewest descendants, indicated by the black check mark.
```

Reviewer: I do feel like if we do that, we should cite Cell Ontology here: https://obophenotype.github.io/cell-ontology/cite/

Up to 10 marker genes are shown per broad cell type.
Only broad cell type annotations present in at least 50 cells across samples in the given diagnosis group are shown.

B. Barplot showing the percentage of each broad consensus cell type annotation across brain and CNS tumors libraries, separated into high-grade (top panel) and low-grade (bottom panel) glioma diagnoses for non-multiplexed libraries from patient tissue samples.

C. Barplot showing all consensus cell types classified as immune cells across brain and CNS tumor libraries, separated into high-grade (top panel) and low-grade (bottom panel) glioma diagnoses for non-multiplexed libraries.
The percentage of immune cells classified as the indicated consensus cell type is shown.
Only libraries comprised of at least 1\% immune cells, based on consensus cell type annotations, are shown.
Specific consensus cell types for myeloid and lymphocyte immune cells are shown, with all other consensus immune cell types included in "other."
Myeloid or lymphocyte immune cell types with fewer than 1000 cells across all libraries are included in "other."

> D. Dot plot as in panel A, but restricted to immune cells across all non-multiplexed libraries from brain and CNS tumors and showing consensus rather than broad cell types showing expression of cell-type-specific marker genes , considering only immune cells.

Reviewer: ```suggestion
D. Dot plot as in panel A, but restricted to immune cells across all non-multiplexed libraries from brain and CNS tumors and showing consensus rather than broad cell types showing expression of cell-type-specific marker genes, considering only immune cells.
```
